### ISSUE TITLE
Exiting CLI doesn't reboot cleanly

### ISSUE THREAD

**Issue by richard-scott** [betaflight/betaflight-configurator#690]
When exiting the CLI, the GUI crashes. 
It used to disconnect and reconnect nicely, but now it doesn't.

I'm using the codebase from commig  a976b88 on the 15th October.

![2017-10-14 16_48_58-betaflight - configurator](https://user-images.githubusercontent.com/5149834/31583388-afa10b3a-b192-11e7-8c50-cb58727fca89.png)

**Comment by richard-scott** [comment]
It's the same when saving blackbox settings too:
![2017-10-15 10_19_37-betaflight - configurator](https://user-images.githubusercontent.com/5149834/31583402-fb001486-b192-11e7-88d4-ef70869b50ed.png)

**Comment by mikeller** [comment]
Try pulling the latest configurator - there was a bug in the disconnection code, it was fixed in #688 / #689.

**Comment by richard-scott** [comment]
My Laptop pulls down the latest codebase from master every time it boots. I've just booted, and tried now and still get the same. All I did this time was connect to the FC (an SP Racing F3 Evo), click Ports, and click Save and Reboot and got this in the screen shot.

![2017-10-15 19_33_25-betaflight - configurator](https://user-images.githubusercontent.com/5149834/31587798-d3d1f5ee-b1df-11e7-8773-ccf0b99428cd.png)
